### ISSUE TITLE
Accessibility - Screen Readers

### ISSUE THREAD

**Issue by forrestwilkins** [forrestwilkins/praxis#100]
All pages should be accessible to screen readers.
- [Accessibility in React - Mozilla](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_accessibility)
- [React Accessibility - react.org](https://reactjs.org/docs/accessibility.html)
- [Introduction to Web Accessibility - W3](https://www.w3.org/WAI/fundamentals/accessibility-intro/)
